spinn_front_end_common/interface/interface_functions/buffer_extractor.py

```diff
@@ -15,6 +15,8 @@ def __call__(self, machine_graph, placements, buffer_manager):
         # Count the regions to be read
         n_regions_to_read, recording_placements = self._count_regions(
             machine_graph, placements)
+        if not n_regions_to_read:
```

Comment:
It was causing an issue to go forward from this point... but if you want it to report doing nothing, we can do the check in after creating the progress bar and then end the progress bar immediately.  There *might* be an issue with passing a progress bar 0 but this would have to be checked if you really think a user cares...

Note that this really says that if nothing is recorded don't extract anything, so a user might be a bit confused by seeing progress on extracting no recorded data.

Reply:
what this says to a end user, is if there's no recording data, don't even try to check..... but i see ya point........... unknown if im happy or not leaving it out.......ill drink on it

Reply:
A simple option is to do logger.info("No recorded data to extract") before the return.

Reply:
conflict between GIGO and completeness and explicitness............. am removing my on edgeness due to knowing this is due to said conflict.

happy to be merged.

Reply:
ooooooo i like the idea of the looger.info. that satisifies my itch